style.css

```diff
@@ -0,0 +1,177 @@
+:root {
+  --header-height: 100px;
+}
+
+html {
+  font-family: 'Merriweather', serif;
+  background: white;
+  color:black;
+  margin: 0px;
+  padding: 0px;
+  width: 100%;
+}
+
+body {
+  margin: 0px;
+  margin-top: var(--header-height);
+  padding: 0px;
+  width: 100%;
+}
+
+/* HEADER */
+
+header {
+  background-color: black;
+  color: white;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  top: 0;
+  position: fixed;
+  z-index: 4;
+  padding: 10px 20px 10px 20px;
+  height: var(--header-height);
+}
+
+.logo {
+  width: 150px;
+}
+
+nav a{
+  color: white;
+  font-size: 14px;
+  text-decoration: none;
+  padding: 0px 10px 0px 10px;
+}
+
+nav a:hover {
+  text-decoration: underline;
+}
+
+header h1 {
+  font-size: 30px;
+}
+
+/* BIG NEWS SECTION */
+
+.big-news {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: center;
+  align-items:center;
+  width: 100%;
+  padding: 30px 20px 20px 20px;
+  margin: 0;
+}
+
+.big-news img {
+  width: 90%;
+}
+
+
+.big-news-text {
+  width: 90%;
+  text-align: left;
+}
+
+.big-news h1 {
+  font-size: 25px;
+}
+
+.big-news p {
+  font-size: 20px;
+}
+
+.big-news-text:hover {
+  text-decoration: underline;
+}
+
+/* OTHER NEWS SECTION */
+
+.other-news {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  flex-direction: row;
+  padding: 10px;
+}
+
+.other-news-card {
+  max-width: 300px;
+  padding: 5px;
+  margin: 20px;
+  justify-content: center;
+  /* border: #ccc solid 0.5px; */
+  box-shadow: 0px 8px 10px 0px rgba(46,46,46,1);
+}
```

Comment:
The cards look super good with the box-shadow! Just try to remember to remove unused code